docs/guides/calibration-jobs.mdx

```diff
@@ -9,21 +9,21 @@ description: Describes what calibration jobs are and how often they run
 
 Multiple aspects of the QPU are affected by the many different calibrations that IBM&reg; runs daily and hourly.
 
-### Daily calibrations
+### Device calibrations
 
-These calibrations run once every day and last 30-90 minutes, depending on the size of the QPU. The devices start this daily calibration at midnight ET, with the calibration jobs interleaving one-for-one with jobs from external users. This causes the calibration jobs to spread out across several hours, sometimes more than half a day. They include the following:
+Calibrations for qubit and gate parameters are conducted on an as-needed basis. Regularly scheduled diagnostic monitoring experiments, interleaved with user jobs, determine when calibrations are needed. These calibrations can occur at any time during the day, based on device performance. Benchmark-measuring experiments that are reported to Qiskit are run daily and interleave with user jobs.
 
 *   **Single-qubit calibrations**
 
-The frequency, pulse amplitudes, and “DRAG” parameter of each qubit are calibrated.
+    The frequency, pulse amplitudes, and “DRAG” parameter of each qubit are calibrated on an as-needed basis.
 
 *   **Single-qubit benchmarking**
 
-    The T1/T2 coherence times and measurement error rates are measured, and randomized benchmarking is run in batches of non-adjacent qubits.
+    The T1/T2 coherence times and measurement error rates are measured, and randomized benchmarking is run in batches of non-adjacent qubits. These benchmarks are measured on a daily basis, with the exception of measurement error, which is reported when readout angle calibrations are performed as well. 
 
 *   **Two-qubit gate calibrations and benchmarking**
 
-    The amplitude and phase of each pulse is calibrated, and randomized benchmarking in batches of well-separated gates of similar length is run.
+    The amplitude and phase of each pulse is calibrated, and randomized benchmarking in batches of well-separated gates of similar length is run. These benchmarks are measured on a daily basis.
```

Comment:
This part is a little confusing. Are the two-qubit gates calibrated as needed as well? Add that layer fidelity is also measured daily.